src/com/cft/contactmerge/io/SplitStringUtilities.java

```diff
@@ -3,26 +3,69 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+
 import java.util.*;
 
 public class SplitStringUtilities {
     public static List<String> splitTsvString(String input)
     {
         List<String> x = new ArrayList<String>();
 
+
+
         String testString = input;
-        String _regex = "[\t]";
 
-        String[] _split = testString.split(_regex);
 
-        for(int i = 0;i< _split.length;i++){
-            x.add(_split[i]);
+        String _string = new String("");
+        for(int i = 0;i< testString.length();i++){
+            if(input.charAt(i)=='\t'){
+                x.add(_string);
+                _string = "";
+            }
+            else
+            {
+                _string= _string+input.charAt(i);
+            }
         }
+        x.add(_string);
 
+        //////////Remove white space before any character, and after the last character in the strings.
 
+        List<String> y = new ArrayList<String>();
 
+        for(ListIterator itb = x.listIterator();itb.hasNext();){
+            String strToCheck = itb.next().toString();
 
-        return x;
+            for(int j = 0;j<2;j++) {
```

Comment:
It's recommended not to hardcore the end iterator in a for loop. If it's a primitive data type array, you can use array.length, and if it's an object, you can use object.size().